### ISSUE TITLE
Cannot install via Carthage anymore for 1.1.1

### ISSUE THREAD

**Issue by jefflen** [chrs1885/Capable#47]
## What did you do?
I was trying to install Capable 1.1.1 by using Carthage.

## What did you expect to happen?
Carthage should be able to build Capable as per 1.1.0 and before.

## What happened instead?
I got this error message:

```
Dependency "Capable" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/chrs1885/Capable/issues/new
```

I found that `Capable.xcodeproj` was removed since 11.1.1 in cd49b00d3b9f0dd968524f25d76c8afd877111a1, I believe this is the cause of the issue.

## Setup
* Added this in the `Cartfile`
```
github "chrs1885/Capable"
```
* Run the command
```
carthage update --platform iOS
```

**Comment by chrs1885** [comment]
Exactly, I guess we need to make the Capable schemes shared to support Carthage again. Is this something you think you could contribute, @jefflen?

**Comment by jefflen** [comment]
Yes, I can try and will create a PR.

**Comment by jefflen** [comment]
@chrs1885 I've created a PR for this, could you review it and potentially release a new version?

**Comment by chrs1885** [comment]
@jefflen can you review #49 and let me know if this works for you as well?

**Comment by chrs1885** [comment]
Fixed with #49
